docs/sources/tempo/configuration/grafana-agent/_index.md

```diff
@@ -14,6 +14,12 @@ aliases:
 collector for sending metrics, logs, and trace data to the opinionated
 Grafana observability stack.
 
+{{< admonition type="note">}}
+Grafana Alloy provides tooling to convert your Agent Static or Flow configuration files into a format that can be used by Alloy.
+
+For more information, refer to [Migrate to Alloy]({{< relref "../grafana-alloy/migrate-alloy" >}}).
```

Comment:
Can the relref be update to the current reference style?

Reply:
```suggestion
For more information, refer to [Migrate to Alloy](https://grafana.com/docs/tempo/<TEMPO_VERSION>/grafana-alloy/migrate-alloy).
```